src/views/LoginPage/index.js

```diff
@@ -1,10 +1,18 @@
+import { CognitoUserPool } from "amazon-cognito-identity-js";
+import * as AmazonCognitoIdentity from "amazon-cognito-identity-js";
```

Comment:
`import *` is generally bad practice, and can really slow an app down. Instead, just import the things you need, so replace line 1 with `import { CognitoUserPool, AuthenticationDetails, CognitoUser } from "amazon-cognito-identity-js";`